src/cascadia/TerminalCore/TerminalSelection.cpp

```diff
@@ -346,7 +355,14 @@ void Terminal::ExpandSelectionToWord()
     {
         const auto& buffer = _activeBuffer();
         _selection->start = buffer.GetWordStart(_selection->start, _wordDelimiters);
+        _selection->pivot = _selection->start;
         _selection->end = buffer.GetWordEnd(_selection->end, _wordDelimiters);
+
+        // if we're targetting both endpoints, instead just target "end"
+        if (WI_IsFlagSet(_selectionEndpoint, SelectionEndpoint::Start) && WI_IsFlagSet(_selectionEndpoint, SelectionEndpoint::End))

```

Comment:
👉 `WI_AreAllFlagsSet`

Reply:
I tried 😭. Compiler gave me sass about `&` not being defined or something like that even though this is at the top of the file:
`DEFINE_ENUM_FLAG_OPERATORS(Terminal::SelectionEndpoint);`

Reply:
Huh... that's weird. We do use `WI_AreAllFlagsSet` in TermControl.cpp already.

Reply:
with this specific enum type? If so, yes, thisd should work. If not, we probably need the `DECLARE_ENUM_BOOLEAN_VALUES` thing...

Reply:
Yeah :(

https://github.com/microsoft/terminal/blob/fecccaa257c1910c883d2afe3b340364504493e7/src/cascadia/TerminalCore/TerminalSelection.cpp#L11

I'm curious. Imma try that declare enum bool thing.

Reply:
ugh, yeah I can't find that macro. We'll figure it out later.

Reply:
It's maddening that `WI_AreAllFlagsClear` works perfectly well nary 20 lines above.

Reply:
SIMILAR: https://github.com/microsoft/wil/issues/45

Reply:
This is likely because of ADL failing on our nested type ;P